scripts/verify-replace.sh

```diff
@@ -16,26 +20,57 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    ERRFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-error.XXX)
+    STDOUTFILE=$(mktemp ${TMP_DIR}$(basename ${repo})-output.XXX)
```

Comment:
grouping them under the `BASE_REPO_PATH` would make more sense. and let's also print them. Something like this:
```suggestion
    ERRFILE=$(mktemp ${BASE_REPO_PATH}/$(basename ${repo})-error.XXX)
    STDOUTFILE=$(mktemp ${BASE_REPO_PATH}/$(basename ${repo})-output.XXX)
    echo "Std output file ${STDOUTFILE}"
    echo "Error output file ${ERRFILE}"
```

Reply:
done